infra/terraform/modules/data_access/common/sentry/sentry/sentry.py

```diff
@@ -0,0 +1,26 @@
+'''
+Environment variables:
+ - ENV, e.g. "dev", "alpha", etc...
+ - SENTRY_DSN, Sentry DSN
+'''
+
+import os
+
+from raven import Client as Sentry
+from raven.transport.http import HTTPTransport as SentryHTTPTransport
+
+
+def catch_exceptions(fn):
+    def wrapped(*args, **kwargs):
+        try:
+            fn(*args, **kwargs)
+        except Exception:
+            client = Sentry(
+                os.environ["SENTRY_DSN"],
+                transport=SentryHTTPTransport,
+                environment=os.environ["ENV"],
+            )
+            client.captureException()
+            raise
+
+    return wrapped
```

Comment:
you can control what is exposed using the `__all__` list - this is more effective and explicit than importing from a separate file, because you could still import everything directly from that file. eg:
```python
__all__ = ['catch_exceptions']

def catch_exceptions(fn):
    ...

def something_that_wont_be_exposed():
    ....
```